server/util/db/db.go

```diff
@@ -738,23 +724,56 @@ func ParseDatasource(ctx context.Context, datasource string, advancedConfig *Adv
 	return nil, status.FailedPreconditionError("no database configured -- please specify at least one in the config")
 }
 
-// sqlLogger is a GORM logger wrapper that supresses "record not found" errors.
+// sqlLogger implements GORM's logger.Interface using zerolog.
+//
+// TODO: maybe implement the optional ParamsFilter interface so that we only log
+// parameterized queries.
 type sqlLogger struct {
-	logger.Interface
-	logLevel logger.LogLevel
+	SlowThreshold time.Duration
+	LogLevel      logger.LogLevel
+}
+
+func (l *sqlLogger) Info(ctx context.Context, format string, args ...interface{}) {
+	log.CtxInfof(ctx, format, args...)
```

Comment:
I believe this will lose the original line numbers. gorm has a util function (utils.FileWithLineNum) that it uses to figure out the caller. Maybe you could use that and prepend that to the beginning of the log line?

Reply:
Done 

![image](https://github.com/buildbuddy-io/buildbuddy/assets/2414826/6991f737-16ae-4ffc-9be0-070f3faa786e)